test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -207,6 +207,38 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 	It("should increase cluster size if pending pods are small [Feature:ClusterSizeAutoscalingScaleUp]",
 		func() { simpleScaleUpTest(0) })
 
+	It("Should scale up GPU pool from 0 [Feature:ClusterSizeAutoscalingGpu]", func() {
+		framework.SkipUnlessProviderIs("gke")
+
+		const gpuPoolName = "gpu-pool"
+		addGpuNodePool(gpuPoolName, "nvidia-tesla-k80", 1, 0)
+		defer deleteNodePool(gpuPoolName)
+		framework.ExpectNoError(enableAutoscaler(gpuPoolName, 0, 1))
+		Expect(len(getPoolNodes(f, gpuPoolName)), 0)
+
+		// add deamon set to register drivers
+		// TODO: why gke is not doing this? Which version should be use
```

Comment:
Can you rephrase this into actionable TODO? Something like:
```TODO(<your username>): <what needs to be than and when>```

As for which version, my guess would be matching K8s version, but we should probably check this with accelerators team